ci_framework/roles/libvirt_manager/tasks/attach_interface.yml

```diff
@@ -33,13 +33,13 @@
     - name: Extract networks from XML
       register: _extracted_xml
       community.general.xml:
-        xmlstring: "{{ _domain_xml.get_xml}}"
+        xmlstring: "{{ _domain_xml.get_xml }}"
         xpath: "/domain/devices/interface/source"
         content: "attribute"
 
 - name: Attach new port if needed
   vars:
-    _net_name: "cifmw-{{ network.name}}"
+    _net_name: "cifmw-{{ network.name }}"
```

Comment:
hmmm so here we enforce the prefix. Since you also allow to NOT prefix network, we'll face an issue here. You probably want to not-prefix as well the _net_name?

Reply:
Thank you. I indeed missed this one.

Since the `vars` is defined in the block. Let me move the check at the block level.